javalib/src/main/scala/java/lang/Throwables.scala

```diff
@@ -163,6 +237,18 @@ class Throwable(s: String, private var e: Throwable)
     duplicates
   }
 
+  def setStackTrace(stackTrace: Array[StackTraceElement]): Unit = {
+    if (writableStackTrace) this.synchronized {
+      var i = 0
+      while (i < stackTrace.length) {
+        if (stackTrace(i) eq null)
+          throw new NullPointerException()
```

Comment:
any error message?

Reply:
Sharp eyes,  I believe that NullPointerException supplies its own message. 

When I did the port, I tried to minimize changes the Scala.js original.  I am always
wary of breaking things.   To the best of my memory, that code is preserved from the original. 

Adding custom message text is always a tradeoff: information added vs. the conceptual cost to devos & end users of being non-standard. I usually like to do the best that I
can to describe where an error happened. Mostly because I am likely to hit that error
myself and call down bad thoughts on the devo/

 The standard NPE is pretty clear & explicit. The accompanying stack trace gives the location where it happened.  I do not see what a custom message might add, but my
vision may be limited.